imgproc.yml

```diff
@@ -4,13 +4,18 @@ channels:
   - nodefaults
 dependencies:
   - conda-forge::python==3.7
-  - conda-forge::matplotlib==3.5.3
+  - conda-forge::matplotlib
   - conda-forge::scikit-image==0.19.3
   - conda-forge::numpy==1.21.6
   - conda-forge::ipykernel==6.16.2
+  - conda-forge::igraph==0.10.3
   - conda-forge::pandas==1.3.5 # a dependency of pylattice
   - conda-forge::pillow==9.2.0 # a dependency of pylattice
   - conda-forge::pims==0.6.1 # a dependency of pylattice
+  - conda-forge::igraph==0.10.3 # a dependency of MitoTNT
+  - conda-forge::tqdm==4.65.0 # a dependency of pylattice
+  - conda-forge::sphinx_rtd_theme==1.2.0 # a dependency of pylattice
```

Comment:
```suggestion

```
this dependency probably isn't necessary for running the notebook
it's usually used for creating readthedocs documentation